_getting-started/index.md

```diff
@@ -7,55 +7,117 @@ includeTOC: true
 
 redirect_from: "/getting-started.html"
 ---
+## Try Scala without installing anything 
+You can try Scala, with access to all Scala compilers and 
+all published libraries with [Scastie](https://scastie.scala-lang.org/)
+
+##  Installing Scala
+### Using the Scala Installer (recommended way)
+The Scala Installer based on coursier will ensure that a JVM and standard Scala tools are installed on your system.  
+
+* Download coursier launcher and execute `setup` command
+
+<div class="main-download">
+    <div id="download-step-one">
```

Comment:
I think it could be useful to be able to see the setup instructions for a specific OS in addition to seeing by default the instructions for the OS of the current user. This could be addressed in a separate PR, though.

Reply:
ok. why not.